worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -661,8 +664,17 @@ private[celeborn] class Worker(
     val resourceConsumptionSnapshot = storageManager.userResourceConsumptionSnapshot()
     val userResourceConsumptions =
       workerInfo.updateThenGetUserResourceConsumption(resourceConsumptionSnapshot.asJava)
+    var appRemainingCount = resourceConsumptionAppLimit
```

Comment:
A reasonable approach is to get the TopK items from user resource consumption for all users instead of just some users.